ads/adhese.js

```diff
@@ -44,13 +44,16 @@ export function adhese(global, data) {
   const co = global.document.querySelector('#c');
   co.width = data['width'];
   co.height = data['height'];
-  co.addEventListener('adhLoaded', getAdInfo, false);
+  co.addEventListener('adhLoaded', e => {
```

Comment:
if you changed `getAdInfo(e, global)` to `getAdInfo(global, e)`, then you can do: `co.addEventListener('adhLoaded', getAdInfo.bind(null, global), false)`